src/software/ai/motion_constraint/motion_constraint.h

```diff
@@ -15,7 +15,7 @@ MAKE_ENUM(MotionConstraint,
           FRIENDLY_DEFENSE_AREA,
           // The box around the enemy goal
           ENEMY_DEFENSE_AREA,
-          // The box around the enemy goal, inflated by 0.2 meters for certain situations
+          // The box around the enemy goal, inflated for certain situations
```

Comment:
super nit: Isn't there just a single size for the `INFLATED_ENEMY_DEFENSE_AREA`, it's just larger than the regular defense area by definition (hence the name inflated...)? Or does how much it's inflated actually change?

"inflated for certain situations" makes me think the size changes dynamically, which IIRC isn't true.

Reply:
yes there is a single size for INFLATED_ENEMY_DEFENSE_AREA, but that can change either by being manually tuned or potentially with a dynamic param, so it's better that we don't hard code it in this comment since the comment is likely to go out of date (it's already out of date)

Reply:
Yes I agree with not hardcoding the value in the comment. What I meant to say was that the "inflated for certain situations" part makes it sound like it can change at runtime under certain cases (not from param tuning). I think the comment would be fine / more clear without the last part. Based on the name it's already obvious its an inflated / larger rectangle.

Maybe just "An inflated box around the enemy defense area"?

Reply:
done